amenity-points.mss

```diff
@@ -3181,6 +3115,50 @@
   }
 }
 
+.text-low-priority {
```

Comment:
I think you should use an id based selector here since this styling is only used in one layer.

Reply:
Thanks! I forgot to remove that when the point and poly layers were unified.